core/ArchiveProcessor/Loader.php

```diff
@@ -105,6 +105,7 @@ public function prepareArchive($pluginName)
     private function prepareArchiveImpl($pluginName)
     {
         $this->params->setRequestedPlugin($pluginName);
+        $this->params->setIsPartialArchive(false);
```

Comment:
Normally when configuring Params objects, we create new ones where partial archive is set to `false` to begin with. Here in Loader, however, we reuse Params objects to archive VisitsSummary by itself, and then the rest of the plugins. When reusing, we have to specifically reset this property, or other archives will think they are partial archives.

This is the only place this should be done, so I did not consider changing Params a good idea, since developers should only use it in Archiver instances for a specific use case.